install.sh

```diff
@@ -49,59 +66,46 @@ if psql -lqt | cut -d \| -f 1 | grep -qw $DBNAME; then
 
 else
 
-   #Create Postgresql Pre-reqs
-   sudo postgres <<EOF
-   psql -c "CREATE USER $DBUNAME WITH PASSWORD '$DBPASS' CREATEDB;"
-   createdb -O$DBUNAME -Eutf8 $DBNAME
-   echo "Postgres user and database '$1' created."
+#Create Postgresql Pre-reqs
+sudo su postgres <<EOF
+psql -c "CREATE USER $DBUNAME WITH PASSWORD '$DBPASS' CREATEDB;"
+createdb -O$DBUNAME -Eutf8 $DBNAME
+echo "Postgres user '$DBUNAME' and database '$DBNAME' created."
 EOF
 
 fi
 
-# TODO
-# echo "Creating web server user"
-# sudo useradd $DBUNAME
-
-echo "Creating clone of git repo"
-sudo mkdir -p $APP_PATH
-# sudo chown $DBUNAME:$DBUNAME $APP_PATH # TODO
-# sudo -u $DBUNAME git clone $GIT_PATH $APP_PATH # TODO
-git clone $GIT_PATH $APP_PATH
 
 echo "Installing required application packages...(if needed)"
 
 yes | pip3 install virtualenv
 
-pushd $APP_PATH
+cd $APP_PATH
 if [ ! -d "$APP_PATH/venv" ]; then
-  # sudo -u $DBUNAME python3 -m virtualenv venv # TODO
-  python3 -m virtualenv venv
+  sudo python3 -m virtualenv venv
```

Comment:
This should probably not be sudo either. The permissions will be too strong and the user won't be able to access the venv

Reply:
Resolved it in a different manner... Sudo works fine - see new commit...